src/main/webapp/app/admin/logs/logs.component.ts

```diff
@@ -2,8 +2,13 @@ import { Component, Inject } from 'vue-property-decorator';
 import { mixins } from 'vue-class-component';
 import Vue2Filters from 'vue2-filters';
 import LogsService from './logs.service';
+import RoutesSelectorVue from '@/shared/routes/routes-selector.vue';
```

Comment:
why these changes for this file ?

Reply:
I will reset this change. It is not necessary here, because I created a new component for instance's logs.